src/irr/asset/CImageLoaderBMP.cpp

```diff
@@ -202,42 +202,43 @@ asset::IAsset* CImageLoaderBMP::loadAsset(io::IReadFile* _file, const asset::IAs
 
 	_file->read(&header, sizeof(header));
 	
-	byte BitmapHeader[header.BitmapHeaderSize];
+	std::vector<uint8_t> BitmapHeader;
```

Comment:
use `core::vector` instead (our typedef with out custom allocator as default, defined in include/irr/core/Types.h)